findfiles_test.go

```diff
@@ -0,0 +1,28 @@
+package pusher
```

Comment:
To encourage blackbox testing, I suggest putting tests in _test packages.  So try pusher_test.

Go tools recognize this and work nicely, but it restricts the tests to using the public interface.

Reply:
Done